google-cloud-tasks/synth.py

```diff
@@ -0,0 +1,42 @@
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+
+v1_library = gapic.ruby_library(
+    'tasks', 'v2beta2', artman_output_name='google-cloud-ruby/google-cloud-tasks',
+    config_path='artman_cloudtasks.yaml'
+)
+
+s.copy(v1_library)
```

Comment:
I think we should avoid replacing the Gemfile, gemspec, readme, etc., even for gapic-only. Just replace the code (in lib and test).